labelbox/schema/annotation_import.py

```diff
@@ -266,41 +252,142 @@ def from_name(cls, client: "labelbox.Client", model_run_id: str,
+            name: Name of the import job. Can be used to reference the task later
+            path: Path to ndjson file containing annotations
+        Returns:
+            MALPredictionImport
+        """
+        if os.path.exists(path):
+            with open(path, 'rb') as f:
+                return cls._create_mea_import_from_bytes(
+                    client, project_id, name, f,
+                    os.stat(path).st_size)
+        else:
+            raise ValueError(f"File {path} is not accessible")
 
     @classmethod
-    def create_from_objects(cls, client: "labelbox.Client", project_id: str,
-                            name, predictions) -> "MALPredictionImport":
-        return cls._create_from_objects(client, project_id, name, predictions)
+    def create_from_objects(
+            cls, client: "labelbox.Client", project_id: str, name: str,
+            predictions: List[Dict[str, Any]]) -> "MALPredictionImport":
+        """
+        Create an MAL prediction import job from an in memory dictionary
+
+        Args:
+            client: Labelbox Client for executing queries
+            project_id: Project to import labels into
+            name: Name of the import job. Can be used to reference the task later
+            predictions: List of prediction annotations
+        Returns:
+            MALPredictionImport
+        """
+        data_str = ndjson.dumps(predictions)
+        if not data_str:
+            raise ValueError('annotations cannot be empty')
+        data = data_str.encode('utf-8')
+        return cls._create_mea_import_from_bytes(client, project_id, name, data,
+                                                 len(data))
 
     @classmethod
     def create_from_url(cls, client: "labelbox.Client", project_id: str,
                         name: str, url: str) -> "MALPredictionImport":
-        return cls._create_from_url(client=client,
-                                    parent_id=project_id,
-                                    name=name,
-                                    url=url)
+        """
+        Create an MAL prediction import job from a url
+        The url must point to a file containing prediction annotations.
+
+        Args:
+            client: Labelbox Client for executing queries
+            project_id: Project to import labels into
+            name: Name of the import job. Can be used to reference the task later
+            url: Url pointing to file to upload
+        Returns:
+            MALPredictionImport
+        """
+        if requests.head(url):
+            query_str = cls._get_url_mutation()
+            return cls(
+                client,
+                client.execute(
+                    query_str,
+                    params={
+                        "fileUrl": url,
+                        "projectId": project_id,
+                        'name': name
+                    })["createModelAssistedLabelingPredictionImport"])
```

Comment:
so you dont need to pass in a gql query here?  I guess i just havent seen this syntax yet.

Reply:
The first arg is the query: ```query_str```